i18n/de.json

```diff
@@ -2256,7 +2256,6 @@
   "invite_member.send2": "Einladungen senden",
   "invite_member.sending": " Sende",
   "invite_member.teamInviteLink": "Sie können außerdem Personen über den {link} einladen.",
-  "katex.error": "Konnte ihren Latex-Code nicht kompilieren. Bitte kontrollieren Sie die Syntax und versuchen es erneut.",
```

Comment:
No need to modify anything but `en.json` -- the non-English files are handled automatically by the translator team, and this file will just get overwritten.

Reply:
Good to know :+1: